configuration-example.yaml

```diff
@@ -2,6 +2,12 @@ output:
   path: build
   locations: locations-combined.json
   services: services.json
+aws_elasticsearch:
+  host: example-domain.us-east-1.es.amazonaws.com
+  port: 1234
+  region: us-east-2
```

Comment:
Is our instance running on `us-east-2` or `us-west-2`? If it's running on west, could you change the default example to this?